iOS_SDK/OneSignalSDK/Source/OneSignalHelper.h

```diff
@@ -27,6 +27,7 @@
 
 #import "OneSignal.h"
 #import "OneSignalWebView.h"
+#import "UIApplication+OneSignal.h"
```

Comment:
is this import needed?

Reply:
I added it so that we can easily use the category throughout the SDK. We shouldn't rely on UIApplication's state throughout the SDK we should use this to account for scene based apps